Use the absolute path of dataDir provided by the system
author: Alex Domingo (Vrije Universiteit Brussel)
--- lib/include/portcullis/portcullis_fs.hpp.orig	2020-07-07 10:15:46.026229000 +0200
+++ lib/include/portcullis/portcullis_fs.hpp	2020-07-07 10:18:06.614475000 +0200
@@ -130,13 +130,7 @@
 			// forward as it may seem because we might have installed to a alternate root.  So wind back the 
 			// exec_prefix to get to root (or alternate root) directory.
 			path ep(EXECPREFIX);
-			path root = ep;
-			path altroot = exe_dir.parent_path();
-			while (root.has_parent_path()) {
-				root = root.parent_path();
-				altroot = altroot.parent_path();					
-			}
-			this->dataDir = altroot / customDataDir / "portcullis";
+			this->dataDir = customDataDir / "portcullis";
    	        this->scriptsDir = this->dataDir / "scripts";
 		}
 		else if (exe_dir.stem().string() == "src") {
